### DISCUSSION TITLE
Rhinoceros + V-Ray support

### DISCUSSION THREAD

**Discussion by gioxyer** [vosen/ZLUDA#191] [General]
Does zluda in the future will support Rhinoceros and V-Ray?

**Comment by vosen** [comment]
We might see V-Ray working in the future, but it's not a priority, see the comment here: https://github.com/vosen/ZLUDA/issues/165#issuecomment-1981656622

**Comment by gioxyer** [comment]
Ok thanks, so I need to follow that comment for future updates?
